examples/stats/web_client_monitoring/app.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * An Express application to receive the web metrics and send to Stackdriver
+ * with the opencensus API.
+ */
+
+"use strict";
+
+const express = require("express");
+const assert = require('assert');
```

Comment:
Please remove ```assert```, unused.

Reply:
Assert is used on L36 to check that the project is set. This is needed when running locally but automatically set when running in App Engine.

Reply:
I missed it, please ignore this comment.